src/nlp.jl

```diff
@@ -530,7 +530,7 @@ function MathProgBase.eval_grad_f(d::NLPEvaluator, g, x)
 end
 
 function MathProgBase.eval_g(d::NLPEvaluator, g, x)
-    tic()
+    VERSION < v"0.7-" && tic()
```

Comment:
Why are the `tic()`s version protected but not the `toq()`s? Does this code work?

Reply:
I think the `toq`s are in the other packages no?

The easiest thing to do will be to disable them in those packages also in 0.7, the only other option is to re-implement the feature ourselves.

Reply:
What other packages? I'm confused because both `tic` and `toq` are deprecated, and it's an error to run `toq` without a corresponding `tic`.

Reply:
Hm, I think for some reason I though they were in the solvers or `MathProgBase`.  I'm not sure how it is that they are not being called in tests but `tic` is.  I'll look into it.

Reply:
This is a bit tricky.  I think the only good option here is to re-implement `tic` `toq` in 1.0.  I don't think we can take this out completely as it looks like some of the `MathProgBase` logic may depend on it.  Errors in tests were probably burried in the 1.0 solver errors.